similar/misc/args.cpp

```diff
@@ -105,7 +105,7 @@ static void AppendIniArgs(const char *filename, Arglist &Args)
 		PHYSFSX_gets_line_t<1024> line;
 		while (Args.size() < MAX_ARGS && PHYSFSX_fgets(line, f))
 		{
-			const auto separator = " ";
+			const auto separator = " \t\n\v\f\r";
```

Comment:
You do not need `\n` or `\r` here, since `PHYSFSX_fgets` terminates lines when it sees those.  I doubt users will have `\v` or `\f` in their files either.